kubernetes-cluster-autoscaler.tf

```diff
@@ -36,27 +36,23 @@ EOF
 data "aws_iam_policy_document" "cluster_autoscaler" {
   statement {
     sid = "Read"
-
```

Comment:
Perhaps we should leave changes like these out unless doing actual changes in a given file. On top of that, I personally find the extra line breaks to increase readability.

Reply:
fine with me, I find easier to read that way.